frontend/app/views/fragments/event/stats.scala.html

```diff
@@ -15,24 +15,26 @@
 </div>
 
 @* Event locations *@
-<div class="stat-item">
-    <div class="stat-item__first">
+@if(event.underlying.ebEvent.venue.name && event.underlying.ebEvent.venue.address) {
```

Comment:
I didn't know that Twirl templates had truthiness!

Reply:
How would you have typically made this check for not `None`? I tried this and it worked, but happy to go with a more idiomatic approach.

Reply:
I've actually updated this conditional to use `isDefined` and I've made it an `||` rather than an `&&` as if either of these properties are defined we can render this location information.